github/libp2p.yml

```diff
@@ -3964,15 +3964,49 @@ repositories:
           restrict_dismissals: false
         required_status_checks:
           contexts:
-            - Build and test (--all-features)
-            - Build and test (--benches --all-features)
-            - Build and test (--no-default-features)
-            - Build on WASM (wasm32-unknown-emscripten)
-            - Build on WASM (wasm32-wasi)
             - Check rustdoc intra-doc links
-            - check-clippy
-            - Integration tests
+            - clippy (stable)
+            - Compile on wasm32-unknown-emscripten
+            - Compile on wasm32-unknown-unknown
+            - Compile on wasm32-wasi
+            - Compile on x86_64-apple-darwin
+            - Compile on x86_64-pc-windows-msvc
+            - Compile with select features (mdns tcp dns async-std)
+            - Compile with select features (mdns tcp dns tokio)
+            # - clippy (beta) # New lints landing in beta should not be a blocker for CI.
+            - IPFS Integration tests
             - rustfmt
+            - Test libp2p-autonat
```

Comment:
Do I understand correctly that each time we add a new crate we have to touch this file? I am assuming there is no out-out instead of opt-in for required status checks?

Reply:
By default, status checks are not required so we don't need to touch the file unless we want them to be required!